core/flow/rule_manager.go

```diff
@@ -89,10 +67,10 @@ func onRuleUpdate(rules []*FlowRule) (err error) {
 }
 
 // LoadRules loads the given flow rules to the rule manager, while all previous rules will be replaced.
-func LoadRules(rules []*FlowRule) (bool, error) {
+func LoadRules(rules []*FlowRule) error {
```

Comment:
The pair(bool, error) of return value is required.
bool: indicate whether this Load update the rule map;
error: indicate whether occur error in this load operation.

Reply:
We might add some cache logic to check whether current value is consistent with last update value. It's under discussion.

Reply:
But in existing code logic, it is meaningless. We return it directly with True.

Reply:
Ok, i recover the code for future design.

Reply:
Currently, we return it directly with True.
As comments upper, it's sure that we will add a cache logic, but the exact design hasn't been decided yet. 
In order to keep all logic consistent in xxx rule manager, it's better to use a pair(bool, error) return value.

Reply:
Thanks~